.gitignore

```diff
@@ -43,7 +43,7 @@ tf.plan
 **/node_modules/
 
 # Ignore lambda builds
-**/code/**/dist/
+**/dist
```

Comment:
I am guessing this is just a simplification?

Reply:
It's so the `dist` folder in the `cognito` module gets ignored as well:
https://github.com/cds-snc/forms-terraform/tree/develop/aws/cognito/lambda/cognito_email_sender

I ran into this problem running a dependency install and  `terraform plan` locally.